linker/Mono.Linker/Driver.cs

```diff
@@ -294,6 +296,7 @@ static void Usage (string msg)
 			Console.WriteLine ("                 default is all");
 			Console.WriteLine ("   -x          Link from an XML descriptor");
 			Console.WriteLine ("   -a          Link from a list of assemblies");
+			Console.WriteLine ("   -r          Link from a list of assemblies using public roots");
```

Comment:
@marek-safar Do you mean it can break a lot of code that uses reflection? Sometimes we can make an assumption that a library will only be accessed via public entry points and then we can use this option. We intend to use it to remove dead code from portions of shared libraries.